tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -81,6 +82,9 @@ qemu-tarball:
 qemu-tdx-experimental-tarball:
 	${MAKE} $@-build
 
+qemu-snp-experimental-tarball:
```

Comment:
@fitzthum, we're trying to keep those alphabetically sorted. Mind to move this entry before the qemu-tarball?